doc/rst/usingchapel/portability.rst

```diff
@@ -0,0 +1,48 @@
+
+
+===========
+Portability
+===========
+
+Chapel is a portable language that is designed to run on a wide variety of hardware and distributed network configurations.
+
+Operating Systems
+-----------------
+
+* Linux
+* MacOS
+* Windows (with WSL)
+
+Compute Architectures
+---------------------
+
+* CPU
+
+  * x86
+
+  * ARM
+
+* GPU
+
+  * NVIDIA
+
+  * AMD
+
+Networks
+--------
+
+* Slingshot
+* Aries
+* InfiniBand
```

Comment:
I think it makes sense to bump it to below EFA as its much older. Another way to avoid this problem entirely is to just alphabetize the list

Reply:
I tend to like the "sort by priority/focus/importance" approach for this list over alphabetizing, particularly since alphabetizing puts Aries first and Slingshot last which inverts the order I'd want to them to be listed.  As indicated in the original comment, I'm not offended by where Aries sits, though also wouldn't be surprised if it had been lower on the list due to its age.

One other thing we could consider here would be to brand these (e.g.,  HPE Cray Slingshot, Cray Aries, Amazon EFA), but that doesn't really help with the alphabetization issue.  I'm mentioning it in this round only because I'm realizing that—other than Ethernet and IB—these terms may not be immediately self-evident to most.